Resources/doc/frontend_dropzone.md

```diff
@@ -4,9 +4,9 @@ Use Dropzone in your Symfony2 application
 Download [Dropzone](http://www.dropzonejs.com/) and include it in your template. Connect the `action` property of the form to the dynamic route `_uploader_{mapping_name}`.
 
 ```html
-<script type="text/javascript" src="https://rawgithub.com/enyo/dropzone/master/downloads/dropzone.js"></script>
+<script type="text/javascript" src="https://raw.github.com/enyo/dropzone/master/dist/dropzone.js"></script>
 
-<form action="{{ oneup_uploader_endpoint('gallery') }}" class="dropzone">
+<form action="{{ oneup_uploader_endpoint('gallery') }}" class="dropzone" style="width:200px; height:200px; border:4px dashed black">
```

Comment:
Can you remove the style attribute here? This is an implementation detail IMHO and should not be in the basic documentation.